pygmt/src/surface.py

```diff
@@ -17,9 +17,14 @@
 
 @fmt_docstring
 @use_alias(
+    C="convergence",
+    G="outgrid",
     I="spacing",
+    Ll="lower",
+    Lu="upper",
+    M="maxradius",
     R="region",
-    G="outgrid",
+    T="tension",
```

Comment:
To let people know what tension values to use, could you change L50 to the following:

```python
    indicates the Laplacian operator. Here, :math:`t = 0` gives the "minimum
    curvature" solution. Minimum curvature can cause undesired oscillations and
    false local maxima or minima (See Smith and Wessel, 1990), and you may wish
    to use :math:`t > 0` to suppress these effects. Experience suggests
    :math:`t \sim 0.25` usually looks good for potential field data and
    :math:`t` should be larger (:math:`t \sim 0.35`) for steep topography data.
    :math:`t = 1` gives a harmonic surface (no maxima or minima are possible
    except at control data points). It is recommended that the user preprocess
    the data with :class:`pygmt.blockmean`, :class:`pygmt.blockmedian`, or
    :class:`pygmt.blockmode` to avoid spatial aliasing and eliminate redundant
    data. You may impose lower and/or upper bounds on the solution. These may
    be entered in the form of a fixed value, a grid with values, or simply be
    the minimum/maximum input data values. Natural boundary conditions are
    applied at the edges, except for geographic data with 360-degree range
    where we apply periodic boundary conditions in the longitude direction.
```

This is copied from https://docs.generic-mapping-tools.org/6.4/surface.html and modified to have proper LaTeX math.

Reply:
No problem. Let me try!  BTW, Happy Chinese New Year! :)

Reply:
Happy Chinese New Year :firecracker: to you too (even though it's past 元宵 already, haha).

I see that you're encountering some merge conflicts, do you need help with that?

Reply:
I think I solved this conflicts. Can you help me check this?

Reply:
Yep, the extra file seems to be deleted now, and the test and docs build at https://pygmt-dev--2321.org.readthedocs.build/en/2321/api/generated/pygmt.surface.html are looking ok!